.travis.yml

```diff
@@ -0,0 +1,23 @@
+# Force OS X.
+language: objective-c
+os: osx
+env: OSX=10.10
+osx_image: xcode7
+rvm:
+  - system
+matrix:
+  include:
+    - env: OSX=10.9
+      osx_image: beta-xcode6.2
+      rvm: system
+
+before_install:
+  - sudo rsync -az "$TRAVIS_BUILD_DIR/" /usr/local/
+  - export TRAVIS_BUILD_DIR="/usr/local"
+  - cd /usr/local
+  - if [ -f ".git/shallow" ]; then git fetch --unshallow; fi
```

Comment:
Yeh, we can always remove it if necessary (or have it done just by `test-bot` for the bottle job), maybe. Will leave it as-is for now